src/OneExplorer.ts

```diff
@@ -65,30 +65,41 @@ export class OneTreeDataProvider implements vscode.TreeDataProvider<OneNode> {
     }
   }
 
-  // TODO(dayo): enable refresh command
   refresh(): void {
     this._onDidChangeTreeData.fire();
   }
 
   getTreeItem(element: OneNode): vscode.TreeItem {
```

Comment:
This command is invalidated as it tries to open every tree nodes. I will implement this properly when supporting open file command.

Reply:
I think this changes is not related to this PR. Can you make a separated PR ?

Reply:
I am worried if I have to make real many prs then.. but I will try.

Reply:
https://github.com/Samsung/ONE-vscode/runs/6222724691